examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
<p>SSN + CELoss + Avg Pool is missing :(</p><p> </p><p>Test Accuracy is the same of SNN with MSE and CE ? I would expect the Accuracy of SNN + CE to match the one of FSS + CE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<blockquote>SSN + CELoss + Avg Pool is missing </blockquote><p>nearly there :) </p>

Reply:
<blockquote>I would expect the Accuracy of SNN + CE to match the one of FSS + CE</blockquote><p>Re-training yielded 0.14 so all good. :+1</p>